libraries/botbuilder/src/inspectionMiddleware.ts

```diff
@@ -34,14 +34,13 @@ class TraceActivity {
         };
     }
 
-    public static fromState(botState: BotState, turnContext: TurnContext, name: string, label: string): Partial<Activity> {
-        var obj = botState.get(turnContext);
+    public static fromState(botState: BotState): Partial<Activity> {
```

Comment:
You changed the signature here. In C# this would be a breaking change. 

IS this something we could avoid?

Reply:
this in 4.4 this is essentially a cherry-pick